lib/prometheus/business_collector.rb

```diff
@@ -18,12 +18,12 @@ def initialize
       'client_accounts_with_hosts',
       'Client accounts with 1 or more hosts (excludes Red Hat)'
     )
-    @total_accounts_with_huge_policies = PrometheusExporter::Metric::Gauge.new(
-      'total_accounts_with_huge_policies',
+    @total_accounts_with_50plus_policies = PrometheusExporter::Metric::Gauge.new(
+      'total_accounts_with_50plus_policies',
```

Comment:
This makes it sound like the number of accounts that have 50 or more policies. Could we specify it as `total_accounts_with_50plus_hosts_per_policy` or similar instead?

Reply:
I guess this naming is getting out of hand already :see_no_evil:

Reply:
It doesn't matter how long the metric/variable name is - it should be as easy to understand as possible from within Grafana.